scripts/aznfs_install.sh

```diff
@@ -92,6 +160,19 @@ canonicalize_distro_id()
     echo "$distro_lower"
 }
 
+# Function to canonicalize boolean values
+canonicalize_boolean() 
+{
+    local value=$(echo "$1" | tr '[:upper:]' '[:lower:]')
+    
+    if [ "$value" == "true" ] || [ "$value" == "false" ]; then
+        echo "$value"
+    else
+        # If set anything other than boolean values, set false.
+        echo "false"
+    fi
+}
+
```

Comment:
we don't need this, we can do a case-insensitive compare 

# Convert to lowercase for easy comparison later.
AUTO_UPDATE_AZNFS=${AUTO_UPDATE_AZNFS,,}